beetmoverscript/task.py

```diff
@@ -62,7 +62,7 @@ def get_task_action(task, script_config):
         messages.append("Only one action type can be used")
 
     action = actions[0]
-    if action not in script_config['actions']:
+    if action not in script.action_map:
```

Comment:
As a constant, `action_map` should probably be in all caps... not your bug but might be a good fix.